### PR TITLE
Show pricing plan and limits

### LINKED ISSUE #631: Show users their pricing plan & limits
We should show our users that they are API call limited. Get the total API calls made in the last 30 days for the organisation (as per the Org usage graph) and show it in the top banner if the calls are within 30% of the limit, so:

"45k out of 50k API calls" with a CTA to upgrade

pls it should be different from other buttons

image

### PR DESCRIPTION
**Pull request by novakzaballa**
Thanks for submitting a PR! Please check the boxes below:

- [X] I have run [`pre-commit`](https://docs.flagsmith.com/deployment/locally-api#pre-commit) to check linting
- [X] I have filled in the "Changes" section below?
- [ ] I have filled in the "How did you test this code" section below?

## Changes

- Updated the "upgrade" button to show Api calls in 30 days/max api  calls in the subscription, when only 30% left

## How did you test this code?

_Please describe._

### CODE REVIEW

File: frontend/common/utils/format.js [hunk 1]
```diff
@@ -112,6 +112,17 @@ const Format = {
     return value
   },
 
+  formatNumber(number) {
```
Comment (gagantrivedi) [inline frontend/common/utils/format.js]: What does this do? Can we add a comment or something?

**Review by gagantrivedi — DISMISSED** [review]

File: frontend/common/utils/format.js (hunk 1 above)
Comment (novakzaballa) [reply frontend/common/utils/format.js]: The function was renamed, and I added a comment. Also, I changed the behavior a bit.

**Comment by kyle-ssg** [comment]
@dabeeeenster @matthewelwell  a couple of product questions based on this PR

- The PR adjusts the upgrade nav to show regardless of their plan, I'm not sure we'd want to do this? Especially with auto seats this seems more relevant to startup / free?
- The PR adds a usage message based on <30% remaining, I guess maybe this is down the line but I'd have thought that the velocity of usage is more important than the amount remaining. E.g. 50% usage in the first 2 months

**Comment by dabeeeenster** [comment]
@kyle-ssg 

point 1: I think we would want to show if people are near their limit regardless of their plan. 

point 2: Given that the data is from the last 30 days, I think current data is fine.

**Review by kyle-ssg — APPROVED** [review]

**Pull request merged**